File: exercicios/1017/gc.cs
//1016
using System;

namespace exercicios{

    class dp{
        static void Main(string[] args){
            double ql, v, th;
            th = Convert.ToDouble(Console.ReadLine());
            v = Convert.ToDouble(Console.ReadLine());
            ql = (th * v) / 12.0;
            Console.WriteLine( String.Format("{0:0.000}", ql));
        }
    }
    
}

File: exercicios/1017/gc.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val th : Double = (scala.io.StdIn.readLine()).toDouble
        val v : Double = (scala.io.StdIn.readLine()).toDouble
        var ql : Double = (th * v) / 12.0        
        println(f"$ql%.3f")      
    }

}